utils/_context/_scenarios/__init__.py

```diff
@@ -453,13 +453,18 @@ def all_endtoend_scenarios(test_object):
             "DD_APPSEC_ENABLED": "false",
             "DD_TRACE_HTTP_CLIENT_ERROR_STATUSES": "200-201,202",
             "DD_SERVICE": "service_test",
+            "DD_TRACE_Kafka_ENABLED": "false",
         },
+        include_kafka=True,
         doc="",
         scenario_groups=[ScenarioGroup.ESSENTIALS],
     )
 
     tracing_config_nondefault_3 = EndToEndScenario(
-        "TRACING_CONFIG_NONDEFAULT_3", weblog_env={"DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING": "false"}, doc="",
+        "TRACING_CONFIG_NONDEFAULT_3",
+        weblog_env={"DD_TRACE_HTTP_CLIENT_TAG_QUERY_STRING": "false", "DD_TRACE_Kafka_ENABLED": "true"},
```

Comment:
We should leave a comment stating that kafka is being used to validate  DD_TRACE_<INTEGRATION>_ENABLED configurations. There's nothing special about kafka, it just an integration that is consistent across all libraries (🤞).

Reply:
It's not present in PHP though.

Reply:
Yeah the plan was to use a different existing endpoint and env var for PHP since is the only one left out.